### ISSUE TITLE
Being able to change datarange for insights without editing them

### ISSUE THREAD

**Issue by JorienDialog** [PostHog/posthog#23682]
## Is your feature request related to a problem?

I've created multiple insights (line charts and funnels). To compare the numbers I want to switch between different periods. In this moment I need to edit the insight to change this, this feels very sensitive to errors (mistakenly changing something else). 

## Describe the solution you'd like

To be able to change the date range when viewing insight results (separately or on dashboards).

### Debug info

```shell
Session: https://us.posthog.com/project/sTMFPsFhdP1Ssg/replay/0190a716-1568-7455-8470-2a0c8719d0aa?t=1338 (at https://eu.posthog.com/project/13808/dashboard/68594#panel=support%3Asupport%3A%3A%3Afalse)
Admin: http://go/adminOrgEU/018c90e9-c418-0000-80b8-1b974bf969f7 (Project: 13808)
Sentry: http://go/sentryEU/13808
```

**Comment by MarconLP** [comment]
Hey @JorienDialog, could you share a bit more context about what you are trying to achieve? It is already possible to modify the timeframe without opening the insight edit screen.

https://github.com/user-attachments/assets/2e21848f-4019-4438-9f37-2e4774d507a4

**Comment by MarconLP** [comment]
Correct - It's not possible to change the dashboards' timeframe without saving the change. Is that what you are looking to do?

**Comment by JorienDialog** [comment]
Yes, that is correct. So to be sure: if I change the date range on a
dashboard and confirm it, will that overwrite it for all insights as the
default date range?

Sincerely / Met vriendelijke groet,

Jorien van Rijswijk
Working 7.30 - 16.00 on Mon, Tue, Thu & Fri



+31 (0) 307600290

Ondiep-zuidzijde 6
3551 BW Utrecht
www.dialog.nl  <http://www.dialog.nl>

**Comment by MarconLP** [comment]
No - If you apply a timeframe onto a dashboard it will override each insight's timeframe but only on the dashboard. The timeframe on the insight directly (not through a dashboard) will not be affected.
